src/Lykke.AzureStorage/Queue/AzureQueueExt.cs

```diff
@@ -107,16 +119,32 @@ public async Task FinishMessageAsync(QueueData token)
             }
         }
         
-        public async Task<string> PutMessageAsync(object itm)
+        public Task<string> PutMessageAsync(object itm)
+        {
+            return InnerPutMessageAsync(itm, null);
+        }
+        
+        public Task<string> PutMessageAsync(object itm, TimeSpan initialVisibilityDelay)
+        {
+            return InnerPutMessageAsync(itm, initialVisibilityDelay);
+        }
+        
+        public async Task<string> InnerPutMessageAsync(object itm, TimeSpan? initialVisibilityDelay)
         {
             var msg = SerializeObject(itm);
+            
             if (msg == null)
```

Comment:
string.NullOrWhiteSpace?

Reply:
Mad exactly as it has been done earlier. In fact, queue message with empty body is ok.

Reply:
And why do we need to skipp null body?

Reply:
I see that in prev implementation null is also skipped - so this is for compatibility reasons. Agree with curretn version.